### ISSUE TITLE
508-defect-4: Save-in-progress text is not read out all at once

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#54518]
### Point of contact

Josh Kim

### Severity level

4, Minor. Consider fixing or exploring in 2-4 sprints post-launch.

### Details

When the in progress alert is shown on the screen, the text is broken into chunks making a screen reader break what should be a continuous sentence into smaller parts

https://user-images.githubusercontent.com/136959/222552162-2c5fd512-9424-40c0-8b56-af1020d17c60.MP4

### Reproduction steps

1. Log into any user
2. Start any form that supports save in progress
3. On the first page, use the "Finish this application later" link
4. Reload the intro page
5. Start a screen reader and navigate the save in progress alert text

### Proposed solution or next steps

Build the title text as a string, then render it all at once

#### Alternative (bandaid)
Use `role="text"`

### References, articles, or WCAG support

1.
2.
3.
...

### Type of issue

- [X] Screenreader
- [ ] Keyboard
- [ ] Focus
- [X] Headings
- [ ] Color
- [ ] Zoom
- [ ] Semantics
- [ ] Axe-core
- [ ] Something else

**Comment by Mottie** [comment]
While attempting to use `role="text"` an eslint message appeared:

![Elements with ARIA roles must use a valid, non-abstract ARIA role](https://user-images.githubusercontent.com/136959/222771758-bea1761a-8d4e-418f-91d3-a2486a426a37.png)

[jsx-a11y/aria-role rule](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/aria-role.md)

**Comment by Mottie** [comment]
Closing this issue as the current options don't appear to solve the issue. Josh opened this ticket https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1592 with which was can continue discussion and try to find a more global solution.
